CHANGELOG.md

```diff
@@ -7,6 +7,10 @@
 * [#5843](https://github.com/bbatsov/rubocop/issues/5843): Add configuration options to `Naming/MemoizedInstanceVariableName` cop to allow leading underscores. ([@leklund][])
 * [#5843](https://github.com/bbatsov/rubocop/issues/5843): Add `EnforcedStyleForLeadingUnderscores` to `Naming/MemoizedInstanceVariableName` cop to allow leading underscores. ([@leklund][])
 
+### Bug fixes
+
+* [#5966](https://github.com/bbatsov/rubocop/issues/5966): Fix a false positive for `Layout/ClosingHeredocIndentation`. ([@koic][])
```

Comment:
I think it would be beneficial to mention that the false positive happened when heredoc content is outdented compared to the closing.

Reply:
Oops! Thank you for pointing that out. I fixed it. Thanks always!